bigtable-dataflow-import/src/test/java/com/google/cloud/bigtable/dataflowimport/testing/BigtableTableUtils.java

```diff
@@ -0,0 +1,93 @@
+package com.google.cloud.bigtable.dataflowimport.testing;
+
+import com.google.cloud.bigtable.dataflow.CloudBigtableConfiguration;
+import com.google.cloud.bigtable.hbase1_0.BigtableConnection;
+import com.google.cloud.bigtable.dataflowimport.HBaseImportOptions;
+import com.google.common.collect.Sets;
+
+import org.apache.hadoop.hbase.Cell;
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HTableDescriptor;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Admin;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.client.ResultScanner;
+import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.client.Table;
+
+import java.io.IOException;
+import java.util.Set;
+
+/**
+ * Helper for creating, deleting and reading a Cloud Bigtable during tests. Each instance manages
+ * a specific table.
+ */
+public class BigtableTableUtils {
+  private static final int MAX_VERSISON = 5;
+
+  private BigtableConnection connection;
+  private final TableName tableName;
+  private final Admin admin;
+
+  private BigtableTableUtils(BigtableConnection connection, TableName tableName)
+      throws IOException {
+    this.connection = connection;
+    this.tableName = tableName;
+    this.admin = connection.getAdmin();
+  }
+
+  /**
+   * Creates an empty table with a single column family as specified by {@code columnFamily}.
+   * If table already exists, it is removed and recreated.
+   */
+  public void createTable(String columnFamily) throws IOException {
```

Comment:
nit:
createTable() makes more sense to take a table name, no?
I don't know. It is up to you.